server/routes/shared/showReferralView.ts

```diff
@@ -61,6 +61,27 @@ export default class ShowReferralView {
     href: this.presenter.referralOverviewPagePresenter.dashboardURL,
   }
 
+  private readonly insetTextArgs = {
+    html: `${ViewUtils.escape(`You can amend the completion date on this referral.`)}<br>
+          ${ViewUtils.escape(`This will send a notification to the service provider`)}`,
+  }
+
+  get notificationBannerArgs(): NotificationBannerArgs {
+    const html = `<b>Referral changes saved</b>
```

Comment:
if this is to style the text in **bold**, the <strong> tag should be used instead.